.devcontainer/devcontainer.json

```diff
@@ -20,7 +20,8 @@
 	"customizations": {
 		"vscode": {
 			"extensions": [
-				"rvest.vs-code-prettier-eslint"
+				"rvest.vs-code-prettier-eslint",
```

Comment:
I'd argue that as prettier/eslint isn't configured in the template, it doesn't make much sense to add this extension by default